test/fastify-mysql.test.js

```diff
@@ -112,7 +114,7 @@ test('should throw with multiple instances and same name', ({ ok, same, plan })
 
   fastify.ready((errors) => {
     ok(errors)
-    same(errors.message, "fastify-mysql has already been registered with name 'first_db'")
+    same(errors.message, 'fastify-mysql has already been registered with name \'first_db\'')
```

Comment:
Why do you replace `"` with `'`?

Reply:
I think that happened with the 
```
standard --fix
```
command

Reply:
Uh ok, that's good!